client/my-sites/themes/theme-activation-confirmation-modal.jsx

```diff
@@ -102,7 +123,7 @@ class AutoLoadingHomepageModal extends Component {
 					},
 					{
 						action: 'activeTheme',
-						label: translate( 'Yes, Activate %(themeName)s', {
+						label: translate( 'Yes, activate %(themeName)s', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.